kuma/core/views.py

```diff
@@ -56,3 +58,11 @@ def rate_limited(request, exception):
     response = render(request, '429.html', status=429)
     response['Retry-After'] = '60'
     return response
+
+
+@shared_cache_control(s_maxage=60 * 60 * 24 * 30)
+def humans_txt(request):
+    """We no longer maintain an actual /humans.txt endpoint but to avoid the
+    sad 404 we instead now just encourage people to go and use the GitHub
+    UI to see the contributors."""
+    return HttpResponse("See https://github.com/mdn/kuma/graphs/contributors\n")
```

Comment:
I was thinking we should make this a 301 redirect. What do you think?

Reply:
I prefer this because it's less disruptive and also explains the transition for those that might care.